victorops/data_source_victorops_rotations.go

```diff
@@ -0,0 +1,105 @@
+package victorops
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceRotations() *schema.Resource {
+	return &schema.Resource{
+		Description: "Get a team's rotations. Note: Rotations are read-only via the API.",
+		ReadContext: dataSourceRotationsRead,
+
+		Schema: map[string]*schema.Schema{
+			"team_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The slug/ID of the team.",
+			},
+			"rotations": {
```

Comment:
please check the rotations data model, take reference from here 
https://plportal.victorops.com/api-docs/#!/Rotations/get_api_public_v2_team_team_rotations, shift should include mask, and members should be shift members